### ISSUE TITLE
Requests not accepted when puttong quotes aorund charset

### ISSUE THREAD

**Issue by jvanheeswijk** [CoreWCF/CoreWCF#1027]
I'm trying to send a SOAP request to a webservice of ours that uses CoreWCF.
In create the request like this:
`HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
request.Method = "POST";
string action = ...;
request.Headers.Add("SOAPAction", action);
string soapMessage = ...;
string rawXml = $@"<soapenv:Envelope xmlns:soapenv=""http://schemas.xmlsoap.org/soap/envelope/"" xmlns:xsi=""http://www.w3.org/1999/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/1999/XMLSchema"">
    <soapenv:Header />
    <soapenv:Body>
        {soapMessage}
    </soapenv:Body>
</soapenv:Envelope>";

using StreamWriter writer = new StreamWriter(request.GetRequestStream());
writer.Write(rawXml);`

Then I try to send it:
`HttpWebResponse wr = (HttpWebResponse) request.GetResponse();`

That gives me the following list of exceptions:
- HttpRequestException: An error occurred while sending the request.
- IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
- SocketException: An existing connection was forcibly closed by the remote host.

I tried explicitly specifying the content type:
`request.ContentType = "text/xml;charset=\"utf-8\"";`
Still the same errors.

Then I removed the quotes around charset:
`request.ContentType = "text/xml;charset=utf-8";`
And then it works.

To summarize: CoreWCF doesn't seem to accept requests when there are quotes around the charset.
